tests/filesystem/clean_path_test.php

```diff
@@ -1,54 +0,0 @@
-<?php
```

Comment:
I think we should have both. The new one to test the feature and the old one to guarantee we don't break anything. But it does not really matter here tbh